tests/acceptance/conftest.py

```diff
@@ -140,7 +140,7 @@ def glue_columns():
     ]
 
 
-@pytest.fixture(scope="session", autouse=True)
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
I had an issue here when running the acceptance tests. They ran in 64 minutes, and everything running on the last 4 failed because the creds were expired. So I decided to move the cognito token fetch from session to module in order to reduce the probability of that happening again.